kontrakter/dom/begjaeringDom/1.0/begjaeringDom.schema.json

```diff
@@ -0,0 +1,699 @@
+    },
+    "dokumenter": {
+      "type": "array",
+      "description": "Det vil alltid være minst 1 dokument som er begjæringsdokumentet, men det skal også legges ved siktelse ...",
+      "items": {
+        "$ref": "#/definitions/dokument"
+      },
+      "minItems": 1
+    }
+  },
+  "required": [
+    "forsendelse",
+    "begjaeringDom",
+    "hovedStraffesaksnummer",
+    "dokumenter"
+  ],
+  "additionalProperties": false,
+
+  "definitions": {
+    "begjaeringDom": {
+      "type": "object",
+      "description": "Begjæringen med basisdata. Detaljer i egne meldinger for siktelse/tiltale og påstand om straff",
+      "properties": {
+        "kravId": {
+          "$ref": "#/definitions/GUID",
+          "description": "Krav om avgjørelse til domstolen, tingretten ønsker avgjørelse (dom tilbake)"
+        },
+        "begjaeringstype": {
+          "type": "string",
+          "description": "Hvilken sak skal dette være, se readme for forklaring. Kun TILSTAAELSE i første omgang.",
+          "enum": [
+            "TILSTAAELSE",
+            "TILSTAAELSE_FORERETT",
+            "TILSTAAELSE_VARETEKT",
+            "MEDDOMSSAK"
+          ]
+        },
+        "ansvarligPaatalejurist": {
+          "$ref": "#/definitions/paataleJurist",
+          "description": "Ansvarlig påtalejurist på straffesaken, kanskje ikke den som har skrevet begjæringen"
+        },
+        "siktedePersoner": {
+          "type": "array",
+          "description": "Alle siktede personer som skal til retten, det vil alltid være bare en person i tilståelsessaker, kobling til basissaker og straffesaker i siktelseTiltale",
+          "items": { "$ref": "#/definitions/siktetPerson" }
+        },
+        "siktedeForetak": {
+          "type": "array",
+          "description": "Alle siktede foretak som skal til retten, kobling til basissaker og straffesaker i siktelseTiltale",
+          "items": { "$ref": "#/definitions/siktetForetak" }
+        },
+        "fornaermedePersoner": {
+          "type": "array",
+          "description": "Alle fornærmede personer, kobling til straffesaken i siktelseTiltale",
+          "items": { "$ref": "#/definitions/personRelatert" }
+        },
+        "fornaermedeForetak": {
+          "type": "array",
+          "description": "Alle fornærmede foretak, kobling til straffesaken i siktelseTiltale",
+          "items": { "$ref": "#/definitions/foretak" }
+        },
+        "vitner": {
+          "type": "array",
+          "description": "Alle vitner med informasjon om hvilke straffesaker de er vitne i",
+          "items": { "$ref": "#/definitions/vitneStraffesak" }
+        }
+      },
+      "required": [
+        "kravId",
+        "siktedePersoner",
+        "siktedeForetak",
+        "fornaermedePersoner",
+        "fornaermedeForetak",
+        "ansvarligPaatalejurist",
+        "vitner"
+      ],
+      "additionalProperties": false
+    },
+    "adresse": {
```

Comment:
Eksempel for å vis mulighetene:Her er det mulig å støtte flere typer på ved å definere regler basert på f.eks land . At det bare er påkrevd visse deler for andre enn norge og omvendt.

```json
{
  "type": "object",
  "properties": {
    "country": { "type": "string" },
    "street_address": { "type": "string" },
    "city": { "type": "string" },
    "state": { "type": "string" },
    "postal_code": { "type": "string" },
    "province": { "type": "string" }
  },
  "required": ["country", "street_address", "city", "postal_code"],
  "allOf": [
    {
      "if": {
        "properties": { "country": { "const": "USA" } }
      },
      "then": {
        "required": ["state"]
      }
    },
    {
      "if": {
        "properties": { "country": { "const": "Canada" } }
      },
      "then": {
        "required": ["province"]
      }
    }
  ]
}
```

Reply:
For personadresser ønsker vi å beholde det så enkelt som nå. Vi tar med oss forslaget ditt når vi oppdaterer gjerningssted (som også er adresse ++) i ny versjon av siktelseTiltale.